sys/kern/subr_intr.c

```diff
@@ -89,6 +89,16 @@
 
 #define	INTRNAME_LEN	(2*MAXCOMLEN + 1)
 
+/*
+ * Archs may define multiple roots with INTR_ROOT_NUM to support different kinds
+ * of interrupts (e.g. arm64 FIQs which use a different exception vector than
+ * IRQs).
+ */
+#if !defined(INTR_ROOT_NUM)
+#define INTR_ROOT_IRQ	0
+#define INTR_ROOT_NUM	1
+#endif
```

Comment:
Is the number of root PICs being constant a good choice?  Perhaps this should instead be dynamic?